bpf/README.md

```diff
@@ -0,0 +1,69 @@
+# BPF 101
+
+If you're reading this, you were probably tasked with writing a BPF code.
+This document will help you get started.
+
+#### Before you start here are some useful links:
+
+https://docs.kernel.org/bpf/ - BPF documentation
+https://github.com/aquasecurity/libbpfgo - Go bindings for libbpf
+https://github.com/iovisor/bcc - BPF Compiler Collection (a lot of examples)
+https://github.com/cilium/cilium - BPF-based networking, security, and observability (more examples)
+https://man7.org/linux/man-pages/man7/bpf-helpers.7.html - BPF helpers documentation
+
+## Environment setup:
+
+We support BPFs only on ARM64 and x86_64 architectures.
+
+1. VM with Ubuntu 20.04+ (or any other Linux distro with the kernel 5.8+ - Docker doesn't work for many BPF features)
+2. Install `clang` and `llvm` (required for compiling BPF code)
```

Comment:
Does `llvm` needs to be explicitly installed, or is whatever clang pulls good enough?

Reply:
`clang` should be enough. I'll fix that.